src/core/lib/transition-manager.js

```diff
@@ -155,6 +160,20 @@ export default class TransitionManager {
     this.state = DEFAULT_STATE;
   }
 
+  // Update angle props so they take shortest interpolation path.
+  _updateAngles(startViewport, endViewport) {
+    const angularProps = ['longitude', 'bearing'];
+    angularProps.forEach((key) => {
+      if (Number.isFinite(startViewport[key])) {
+        assert(Number.isFinite(endViewport[key]));
+        if (Math.abs(endViewport[key] - startViewport[key]) > 180) {
```

Comment:
Should we have a `findShortestAngle` helper? Is that what this is doing? Or at least a comment?

Reply:
I have a comment for `_updateAngles` method.